src/Form/AdminSettingsForm.php

```diff
@@ -377,6 +377,16 @@ private function buildAdvancedForm() {
       '#tree' => TRUE,
       '#open' => FALSE,
     ];
+    $form['bootstrap_mode'] = [
+      '#type' => 'radios',
+      '#title' => t('Bootstrap Mode'),
+      '#description' => t('How would you like to bootstrap Lift? Setting manul allows you to run additional scripts (useful if collecting and sending additional data or forcing an opt-in for users) but you must <a href="https://docs.acquia.com/lift/api/javascript/personalize/" target="_blank">run the personalize() method in code</a> or Lift will not bootstrap.'),
+      '#default_value' => $advanced_settings['bootstrap_mode'],
+      '#options' => [
+        'auto' => t('Automatic'),
```

Comment:
Let's change `Automatic` to `Auto` to keep consistency between D8 and D7 modules.